.snapcraft/resources/startRocketChat

```diff
@@ -20,6 +20,10 @@ function start_rocketchat {
         export MONGO_URL="$(snapctl get mongo-url)"
         export MONGO_OPLOG_URL="$(snapctl get mongo-oplog-url)"
 	export Accounts_AvatarStorePath=$SNAP_COMMON/uploads
+	siteurl="$(snapctl get siteurl)"
+        if [ -n "$siteurl" ]; then
+	  export OVERWRITE_SETTING_Site_Url=$siteurl
```

Comment:
It would be great to see this expanded to allow configuration of any rocketchat setting from snap.
Something similar to that performed by the ansible rocketchat role here: https://github.com/RocketChat/Rocket.Chat.Ansible/commit/81060e965f6dd538aafb3abd4849f599cc04ba97
(See also https://github.com/RocketChat/Rocket.Chat.Ansible/issues/67)

Reply:
Should be right after that, can drop any file with env variables in $SNAP_COMMON directory and they will be exported

Reply:
@nickbroon the problem we run into is snap settings are very picky.  So we can't even use the exact same syntax.  If we could we could easily let any setting be set.  But we would either have to have a predefined list or a very reliable way of converting between one format and another.

the environment variable file in the root is the current way we're going to recommend doing this.